stats/stats/src/blockscout_indexing.rs

```diff
@@ -0,0 +1,66 @@
+use crate::kv_storage;
+use blockscout_db::entity::blocks;
+use migration::DbErr;
+use sea_orm::{prelude::*, sea_query::Expr, DatabaseConnection, FromQueryResult, QuerySelect};
+
+const MIN_BLOCK_KEY: &str = "min_block";
+
+pub async fn is_blockscout_indexing(
+    blockscout: &DatabaseConnection,
+    db: &DatabaseConnection,
+) -> Result<bool, DbErr> {
+    let min_block_blockscout = get_min_block_blockscout(blockscout).await?;
+    let min_block_saved = get_min_block_saved(db).await?;
+    tracing::info!(
+        min_block_blockscout = min_block_blockscout,
+        min_block_saved = min_block_saved,
+        "checking min block in blockscout database"
+    );
+    if min_block_blockscout < min_block_saved {
+        set_min_block_saved(db, min_block_blockscout).await?;
```

Comment:
we should update saved value after we updated all the charts, or this could happen:

1. we see, that blockscout is indexing, and want to update
2. first we update the min block in our kv
3. then the service crashes for some reason
4. we restart the service and see, that blockscout doesn't have anything new
5. but in reality we didn't updated the last blocks, so we have wrong state